### ISSUE TITLE
Package samples separately from the library.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#155]
```
The suggested installation for the library is easy_install/pip which discards 
the samples. Break them apart into separate downloads.
```

Original issue reported on code.google.com by `jcgregorio@google.com` on 20 Jun 2012 at 1:55
- Blocking: #154

**Comment by GoogleCodeExporter** [comment]
Original comment by `jcgregorio@google.com` on 20 Jun 2012 at 1:55
- Now blocking: #154

**Comment by GoogleCodeExporter** [comment]
```
CL: http://codereview.appspot.com/6304106/
```

Original comment by `jcgregorio@google.com` on 20 Jun 2012 at 4:02

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision a1c7910006c8.
```

Original comment by `jcgregorio@google.com` on 20 Jun 2012 at 4:16
- Changed state: **Fixed**
